src/open_petro_elastic/material/span_wagner/tables/lookup_table.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from scipy.interpolate import RectBivariateSpline
+
+
+def generate_lookup_table(func, x, y, filename):
+    grids2d = np.meshgrid(x, y, indexing='ij')
+    grids = [g.flatten() for g in grids2d]
+    z = func(*grids)
+    z_grid = z.reshape(grids2d[0].shape)
+    np.savez(filename, z_grid=z_grid, x=x, y=y)
```

Comment:
Should we use `savez_compressed` instead of `savez` to store it as compressed file in Github ? Not sure if `np.load()` is able to deal with the compressed npz file directly or if it need un-zipping first

Reply:
`np.load` supports both, but I am not sure there is anything to gain by using `np.savez_compressed`. It seems the savings are only around 5-10% (for the current settings).

Reply:
sure, then it can probably stay as it is ;)